geode-protobuf/src/test/java/org/apache/geode/internal/protocol/protobuf/operations/GetAllRequestOperationHandlerJUnitTest.java

```diff
@@ -78,9 +76,9 @@ public void setUp() throws Exception {
 
   @Test
   public void processReturnsExpectedValuesForValidKeys() throws Exception {
-    Result<RegionAPI.GetAllResponse> result = operationHandler.process(serializationServiceStub,
-        generateTestRequest(true, false),
-        new MessageExecutionContext(cacheStub, new NoOpAuthorizer(), new NoOpProtobufStatistics()));
+    Result<RegionAPI.GetAllResponse> result =
+        operationHandler.process(serializationServiceStub, generateTestRequest(true, false),
+            ProtobufTestUtilities.getNoAuthExecutionContext(cacheStub));
```

Comment:
Why would we not have tailored this utilities class to be specific to authentication requests? Also, why would we mix Protobuf message generation and `MessageExecutionContext` creation in the same utility class.

Reply:
Ugh, looks like this is something that leaked out of the handshake work Galen and Sarge had worked on (no idea how that happened).  I'm just going to rename this utility class to ProtobufTestExecutionContext and remove the unused functions.